src/System.Device.Gpio/System/Device/Spi/Devices/UnixSpiDevice.Linux.cs

```diff
@@ -37,9 +37,10 @@ public UnixSpiDevice(SpiConnectionSettings settings)
         public string DevicePath { get; set; }
 
         /// <summary>
-        /// The connection settings of a device on a SPI bus.
+        /// The connection settings of a device on a SPI bus. The connection settings are immutable after the device is created
```

Comment:
this doesn't need summary since it's an internal class (also you can use inheritdoc)

Reply:
same for Windows10SpiDevice

Reply:
you are right, we should probably apply the inheritdoc over all of the driver classes in iot repo. I will do that in a subsequent PR so that this one doesn't look polluted by all those changes.